src/App.tsx

```diff
@@ -1,7 +1,33 @@
-const App = () => (
-  <h1 className="text-3xl font-bold text-red-500 underline text-center">
-    Hello world!
-  </h1>
+import React, { FC } from "react";
+
+import { Routes, Route } from "react-router-dom";
+import ROUTES from "../src/constants/routes/index";
+
+import Login from "./components/pages/Login";
+import Home from "./components/pages/Home";
+import Movies from "./components/pages/Movies";
+import Trending from "./components/pages/Trending";
+import Movie from "./components/pages/Movie";
+
+import PrivateRoutes from "./components/pages/PrivateRoutes";
+import UserProfile from "./components/pages/UserProfile";
+import Favourites from "./components/pages/Favourites";
+
+import NotFound from "./components/pages/NotFound";
+
+const App: FC = () => (
+  <Routes>
+    <Route path={ROUTES.LOGIN} element={<Login />} />
```

Comment:
must be availbale only for user who is not logged in 
Rename Login to Sign In
add Sign Up page route